File: app/src/main/java/cn/make1/myhttptest/utils/TextUtil.java
package cn.make1.myhttptest.utils;

import android.text.TextUtils;

/**
 * Comment: //text工具类
 *
 * @author Jax.zhou in Make1
 * @date 2018/6/13
 * Company:Make1
 * Email:Jax.zhou@make1.cn
 */
public class TextUtil {

    public static boolean checkTextIsEmpty(String text) {
        if (text.length() == 0){
            return true;
        }
        if (TextUtils.isEmpty(text)){
            return true;
        }
        if (text.contains(" ")){
            String newStr = text.replaceAll(" ","");
            return TextUtils.isEmpty(newStr);
        }
        return false;
    }
}

File: app/src/main/java/top/iwill/tinyapp/utils/TextUtil.kt
package top.iwill.tinyapp.utils

import android.text.TextUtils

/**
 * Comment: //text工具类
 *
 * @author Jax.zhou in Make1
 * @date 2018/6/13
 * Company:Make1
 * Email:Jax.zhou@make1.cn
 */
@Suppress("unused")
object TextUtil {

    fun checkTextIsEmpty(text: String): Boolean {
        if (TextUtils.isEmpty(text)) {
            return true
        }
        if (text.contains(" ")) {
            val newStr = text.replace(" ".toRegex(), "")
            return TextUtils.isEmpty(newStr)
        }
        return false
    }
}
